san/sanbase_graphql.py

```diff
@@ -464,6 +464,32 @@ def ohlcv(idx, slug, **kwargs):
     return merged[return_fields]
 
 
+def get_metric(idx, query, slug, **kwargs):
+    kwargs = _transform_query_args(**kwargs)
+
+    query_str = """
+    query_{idx}: getMetric(metric: \"{query}\"){{
+        timeseriesData(
+            slug: \"{slug}\",
+            from: \"{from_date}\",
+            to: \"{to_date}\",
+            interval: \"{interval}\",
```

Comment:
GraphQL has insignificant commas, I would prefer we do not add commas where not needed: https://github.com/graphql/graphql-spec/blob/master/spec/Section%202%20--%20Language.md#insignificant-commas